test/pageobjects/page.js

```diff
@@ -0,0 +1,9 @@
+
+
+module.exports = class Page {
+
+    open (path) {
+        return browser.url(`https://pastebin.com/${path}`)
```

Comment:
if you don't pass `path` parameter your browser will open https://pastebin.com/unedfined